### ISSUE TITLE
AI Abrakhan's cheat CB

### ISSUE THREAD

**Issue by Remerod** [MattTheLegoman/RealmsInExile#273]
To my knowledge Abrakhan was given a cheat CB that was only available to the AI and one of these are a Empire invasion CB. 
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/79544635/151714767-31d05748-ade6-494d-89f8-ae38478a8fd7.png)

</details>

Some people have pointed out that this is even more unfair then Mordor's CB 
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/79544635/151714814-1904339f-87ce-4247-93cf-7f6b18c8a19e.png)

</details>

And well.. i kind of have to agree, Abrakhan is strong but an empire cb that robs everything from you as Bellakar or Harwen is... well there is no comeback from this, do we want this to be a war/enemy where there is no return from? 
I'm eagerly awaiting some feedback on this.

**Comment by JukeJukestar** [comment]
I agree with you fully, infinite empire CBs are quite overpowered. Getting AI Abrakhan to be aggressive is definetly important, but with empires as a target its far too strong.
As a player, the CBs you get normally are good enough, but if a special CB is needed to get the AI going then it should be available to players too. Both infinite Kingdom invasions or infinite forced vassalization CBs could work as an alternative

**Comment by Remerod** [comment]
I think they would be the same amount of aggressive without the empire cb, but i think it's worth checking out

**Comment by Remerod** [comment]
I don't see Abrakhan attacking Bellakar ever in the current set up, given i have cheated as abrakhan to be super big, gotten them to the extend that they can subjugate bellakar, and runned 4-5 games for 10-40 year forward. but they just wont attack bellakar, even some of their subjects did but not themselves. 

![image](https://user-images.githubusercontent.com/79544635/155857825-0112abfe-1fda-433b-8197-3f886a1ad3cf.png)

i'm gonna ask someone else to see if they gotten better luck on it

**Comment by jj248** [comment]
> I don't see Abrakhan attacking Bellakar ever in the current set up, given i have cheated as abrakhan to be super big, gotten them to the extend that they can subjugate bellakar, and runned 4-5 games for 10-40 year forward. but they just wont attack bellakar, even some of their subjects did but not themselves.
> 
> ![image](https://user-images.githubusercontent.com/79544635/155857825-0112abfe-1fda-433b-8197-3f886a1ad3cf.png)
> 
> i'm gonna ask someone else to see if they gotten better luck on it

We can address this in refinements in the future, but the fix specifically for the unfair CB against the player is now resolved, so closing this issue.
